package.json

```diff
@@ -63,7 +63,9 @@
     "lib": "./lib/mongoose"
   },
   "scripts": {
-    "lint": "eslint .",
+    "lint": "npm run lint-ts && npm run lint-js",
```

Comment:
I'd rather avoid this change because I like to run `npm run lint -- --fix` to fix linter issues, and this would break that command. Maybe we keep `lint` script the same as it was, and add `lint-js` and `lint-ts`?